views/js/templates/ProctorManager/status.tpl

```diff
@@ -1,2 +1,2 @@
 {{#if authorized}}{{__ 'Authorized'}} <span class="icon icon-authorize"></span>{{/if}}
-{{#if partially}}{{__ 'Partially authorized'}}{{/if}}
\ No newline at end of file
+<span title="{{testCenters}}">{{#if partially}}{{__ 'Partially authorized'}}{{/if}}</span>
```

Comment:
It could be better if you put the span inside the if, in order to avoid useless span displayed